lib/init-transforms/entry.js

```diff
@@ -0,0 +1,6 @@
+const run = (config, answer) => {
+	config.entry('index').add(answer).end();
```

Comment:
If you are going to return the chain, that's different. Although there isn't much point in returning any config since its all mutable.